extension/app/ts/AddressBook.tsx

```diff
@@ -5,6 +5,7 @@ import Blockie from './components/subcomponents/PreactBlocky.js'
 import { GetAddressBookDataReply, MessageToPopup } from './utils/interceptor-messages.js'
 
 type ActiveFilter = 'My Active Addresses' | 'My Contacts' | 'Tokens' | 'Non Fungible Tokens' | 'Other Contracts'
+const PAGE_SIZE = 100
```

Comment:
What is the purpose of paging the data?  Since we aren't fetching from a remote source, we aren't saving resources or response time by paging the data.  It feels like a single long scrollable list is preferable to paging here.